packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -456,7 +456,7 @@ const MOCK_ACTIVE_SUBSCRIPTIONS = [
   {
     uid: 'a90fef48240b49b2b6a33d333aee9b13',
     subscriptionId: 'sub0.28964929339372136',
-    productName: '123doneProProduct',
+    productId: '123doneProProduct',
```

Comment:
@lmorchard, in the payments server I just did a blanket `s/productName/productId/` and nothing seemed to fail, so...?